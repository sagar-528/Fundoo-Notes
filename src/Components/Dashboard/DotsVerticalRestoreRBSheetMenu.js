import React, { Component } from 'react'
import {View} from 'react-native'
import { Menu, TouchableRipple} from 'react-native-paper'

export class DotsVerticalRestoreRBSheetMenu extends Component {
   
   constructor(props) {
       super(props)
   }
   
    render() {
        return (
            <View style = {{flex : 1}}>
                <TouchableRipple onPress = {this.props.restore}>
                    <Menu.Item icon = "restore" title = "Restore" />
                </TouchableRipple>
                <TouchableRipple onPress = {this.props.deleteForever}>
                    <Menu.Item icon = "delete-forever" title = "Delete forever" />
                </TouchableRipple>
            </View>
        )
    }
}

export default DotsVerticalRestoreRBSheetMenu
