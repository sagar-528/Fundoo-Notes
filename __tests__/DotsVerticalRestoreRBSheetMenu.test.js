import React from 'react';
import {shallow} from 'enzyme';
import DotsVerticalRestoreRBSheetMenu from '../src/Components/Dashboard/DotsVerticalRestoreRBSheetMenu'
import { Menu, TouchableRipple } from 'react-native-paper';

describe('test DotsVerticalRestoreRBSheetMenu', () => {

    it('should match to snapshot', () => {
        const component = shallow(<DotsVerticalRestoreRBSheetMenu />)
        expect(component).toMatchSnapshot();
    })

    it('When menu item in DotsVerticalRBSheetMenu component', () => {
        const component = shallow(<DotsVerticalRestoreRBSheetMenu />)
        expect(component.find(Menu.item)).toHaveLength(2)
        expect(component.find(TouchableRipple)).toHaveLength(2)
    })

})