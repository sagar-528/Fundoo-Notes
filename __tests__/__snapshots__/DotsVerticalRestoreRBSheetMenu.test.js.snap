// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test DotsVerticalRestoreRBSheetMenu should match to snapshot 1`] = `ShallowWrapper {}`;
